src/command_modules/azure-cli-container/README.rst

```diff
@@ -0,0 +1,186 @@
+            az container create -g MyResourceGroup --name MyApp --location westus --image myimage:latest
+            --cpu 1 --memory 1.5 --image-registry-login-server myregistry.com --image-registry-username
+            username --image-registry-password password
+
+Commands to get an Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container show: Show the details of a container group.
+
+    Arguments
+        --name -n           [Required]: The primary resource name.
+        --resource-group -g [Required]: Name of resource group. You can configure the default group
+                                        using `az configure --defaults group=<name>`.
+
+    Global Arguments
+        --debug                       : Increase logging verbosity to show all debug logs.
+        --help -h                     : Show this help message and exit.
+        --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
+                                        Default: json.
+        --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                        information and examples.
+        --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+Commands to tail the logs of a Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container logs: Tail the log of a container group.
+
+    Arguments
+        --name -n           [Required]: The primary resource name.
+        --resource-group -g [Required]: Name of resource group. You can configure the default group
+                                        using `az configure --defaults group=<name>`.
+        --container-name
+
+    Global Arguments
+        --debug                       : Increase logging verbosity to show all debug logs.
+        --help -h                     : Show this help message and exit.
+        --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
+                                        Default: json.
+        --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                        information and examples.
+        --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+Commands to delete an Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container delete: Delete a container group.
+
+    Arguments
+        --name -n           [Required]: The primary resource name.
+        --resource-group -g [Required]: Name of resource group. You can configure the default group
+                                        using `az configure --defaults group=<name>`.
+        --yes -y                      : Do not prompt for confirmation.
```

Comment:
The delete operation in our case will return immediately. So --no-wait option is not really needed.
For create operation, the server response seems missing some info in the http header. Will work on that in a different PR afterwards.